src/Route/RouteCollection.php

```diff
@@ -148,10 +148,8 @@ public function restore(string $name): RouteCollectionInterface
 
     /**
      * @param string|string[] $routeList
-     *
-     * @return static
      */
-    public function clearExcept($routeList): RouteCollectionInterface
+    public function clearExcept($routeList): static
```

Comment:
Is this a BC break ?

Reply:
Oh, same it's final

Reply:
This one might need more thinking, not sure if the change is correct.

Reply:
To me looks better now, and it shouldn't be bc break since its final.